### ISSUE TITLE
Issue with Medkit constantly missing basic features

### ISSUE THREAD

**Issue by WinterrrWarlock** [chrisk123999/chris-premades#329]
Absolutely love your work man, game changing. The issue I have is pretty damn annoying to track down though. I use CPR and Midi-qol to have a lot of automations in my game, and I try to update things as relatively quickly as modules allow me to. I have Foundry V12, CPR 1.0.39, and Midi 11.6.25.1. 

In last weeks session a rogue in my campaign was flanking, had advantage, and was attacking. Sneak Attack should've automatically gone off. It didn't. I was like, hm that's weird, let me check the Medkit. Medkit says "hey everything is good to go!" So i checked MIDI, and all the appropriate settings were turned on. Finally I opened up the Sneak Attack Feature in his character sheet, clicked edit, and THEN looked at the Medkit and lo and behold the automation was set to none. Now, once automations are set by CPR i don't touch them, ever. What's even weirder is that the week previous it worked fine. The only thing that changed was that a new version of CPR had released, and whenever a new one releases I manually click the medkit on each character sheet to make sure they are all updated. 

TLDR: The medkit incorrectly reports that all features are applied even if they arent. Sneak Attack is just one example, but I've had to crawl through each character sheet and check the attacks/spells/features one by one and it's really time consuming XD

It would be nice if it consistently reported the correct information, and I also really miss it printing the "updated" list of things to the chat.

**Comment by chrisk123999** [comment]
The medkit has had some recent bug fixes. Possibly it's tied to that. There's not much I can do with the provided information from here. As far as I know the medkit is currently working correctly.

**Comment by WinterrrWarlock** [comment]
For sure! I know you've been working really hard on it. Is there a specific kind of info that I can give you? I'm happy to provide whatever I can. I absolutely love your work, and it's been a really big inspiration for me to write my own code for my campaign ^_^ one last thing, is there any chance of Gelatinous Cube code coming? I know you just did mimic (and while they both have Pseudopod attacks) it would be dope if Engulf got some love :)

**Comment by chrisk123999** [comment]
The Discord server has a forum for feature requests. Further troubleshooting may be more easily handled on our Discord for your Medkit issue.
